frontend/src/containers/cluster.js

```diff
@@ -121,6 +121,7 @@ class Cluster extends Component {
 
         return (
             <div>
+                <Statistics />
```

Comment:
I agree to the overhaul - doughnuts are great on screenshot, but terrible for quick overview.. I'll try to come up with something more dense asap